### PR TITLE
Make pr comment parsing a python script

### PR DESCRIPTION
**Pull request by Wumpf**
### What

(see title)

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/6579?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/6579?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!

- [PR Build Summary](https://build.rerun.io/pr/6579)
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

**Comment by Wumpf** [comment]
@rerun-bot this is an error

**Comment by Wumpf** [comment]
oh .. right I can't test this because is uses the workflow from `main`. Which is good in a way, but also super annoying

File: scripts/ci/parse_bot_pr_comment.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# List of valid inputs for the bot
+valid_inputs = ["full-check"]
+
+
+comment_body = os.environ.get("GITHUB_COMMENT_BODY", "")
+bot_invocation = re.search(r"@rerun-bot\s+([a-zA-Z\-]*)", comment_body)
+if bot_invocation:
+    command = bot_invocation.group(1)
+    if str(command) in valid_inputs:
+        print(f"command={command}")
```
Comment (jprochazk) [inline scripts/ci/parse_bot_pr_comment.py:27]: I think we should directly write to `GITHUB_OUTPUT` here, using something like this (untested)

```py
def gh_set_output(key: str, text: str) -> None:
    line = f"{key}={text}"
    output_file = os.environ.get("GITHUB_OUTPUT")
    if output_file is not None:
        # append to output_file
        with open(output_file, "a", encoding="utf8") as f:
            f.write(f"{line}{os.linesep}")
    else:
        # print when not running on CI
        print(line)
```

```suggestion
        gh_set_output("command", command)
```

I do a similar thing here for our custom Vercel action:
https://github.com/rerun-io/rerun/blob/ab526f356775ac5fadb4f26742ca215bffe045b8/.github/actions/vercel/util.mjs#L35-L43

Comment (Wumpf) [reply scripts/ci/parse_bot_pr_comment.py:27]: ah, I wasn't sure if that would be exported all the way, somehow assumed that the commands run sandboxed, but that was a different ci system I worked with at some point
👍

**Review by jprochazk — APPROVED** [review]

**Pull request merged**